MonoGame.Framework/Graphics/GraphicsDevice.OpenGL.cs

```diff
@@ -88,11 +88,15 @@ internal void SetVertexAttributeArray(bool[] attrs)
             }
         }
 
-        private void PlatformSetup()
+       private void PlatformSetup()
         {
 #if WINDOWS || LINUX || ANGLE
             GraphicsMode mode = GraphicsMode.Default;
-            var wnd = (Game.Instance.Window as OpenTKGameWindow).Window.WindowInfo;
+
+            var wnd = OpenTK.Platform.Utilities.CreateWindowsWindowInfo(PresentationParameters.DeviceWindowHandle);
```

Comment:
@thefiddler @cra0zy - Is this safe to do for even the default window?

If so the `if (Game.Instance)` case below isn't needed then.